app/src/main/java/org/thoughtcrime/securesms/preferences/Util.kt

```diff
@@ -0,0 +1,31 @@
+package org.thoughtcrime.securesms.preferences
+
+import android.content.ClipData
+import android.content.ClipboardManager
+import android.content.Context
+import android.content.Intent
+import android.widget.Toast
+import network.loki.messenger.R
+import org.session.libsession.utilities.TextSecurePreferences
+
+fun Context.sendInvitation() {
```

Comment:
What kind of invitation is this? To 1-on-1 chat, to join a group or community? Any / all of these?

Reply:
General invitation to download the app.

Reply:
Then perhaps we could have it as  `sendInvitationToUseSession`? It's a few extra chars but it makes it very clear.

Reply:
done

Reply:
Nice <3